crates/engine/tree/Cargo.toml

```diff
@@ -85,3 +85,8 @@ test-utils = [
   "reth-static-file",
   "reth-tracing",
 ]
+optimism = [
+    "reth-primitives/optimism",
+    "reth-provider/optimism",
+    "reth-revm/optimism"
+]
```

Comment:
I really want to avoid this, why do we need this?

Reply:
optimism tests were stalling, let me double check that it wasn't just a flake

Reply:
ok, turns out it was just a flake, will file an issue for the flake and remove this